client/signup/config/steps-pure.js

```diff
@@ -412,6 +412,18 @@ export function generateSteps( {
 			delayApiRequestUntilComplete: true,
 		},
 
+		'domains-with-email': {
+			stepName: 'domains-with-email',
+			providesDependencies: [ 'siteId', 'siteSlug', 'siteUrl', 'domainItem' ],
+			props: {
+				forceHideFreeDomainExplainerAndStrikeoutUi: true,
+				get headerText() {
+					return i18n.translate( 'Choose a domain for your Professional Email' );
+				},
+				includeWordPressDotCom: false,
```

Comment:
```suggestion
				includeWordPressDotCom: false,
				isPlanSelectionAvailableLaterInFlow: false,
```

Reply:
I checked data from the latest 2 months, the "Paid annual plan" sidebar widget doesn't generate any purchases of any product at all. Not only is the widget distracting users from their intention to setup Professional Email, it isn't doing any good in upsell. Let's remove it!

<img width="330" alt="Screen Shot 2022-08-25 at 11 06 08 PM" src="https://user-images.githubusercontent.com/104910361/186809143-15fb6a89-2fcd-4ca2-8665-e3ce30d0a1d6.png">

Reply:
Good catch! But I actually opted for a different solution, which was to just hide the `Choose my domain later` button and keep the text.

My reasoning: the text is a pretty reasonable explainer, even more so since we will be changing the plans step to let users proceed with a free plan. If they do that, the domain will cost money once they reach checkout even if the domains step told them that the domain is `free for the first year`. By keeping this widget, we clarify the conditions for the domain being free. 

Then again, I guess you could argue that the plans page would _also_ probably clarify that the free plan will _not_ include a free domain for the first year. With this clarification, users shouldn't be completely dumbfounded when they reach checkout anyway. See this screenshot for clarification (the actual text would say the opposite, but I've circled the thing I'm talking about):

<img width="1042" alt="image-5" src="https://user-images.githubusercontent.com/1101677/186876654-e2335afc-dc1d-4fc8-b379-42d8e38490d5.png">

I'm open to hiding the entire widget as well, but I think that this approach does add value.

Reply:
Yeah, I'm onboard with this approach!